MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -209,6 +209,13 @@ private string GetMjpegEncoder(EncodingJobInfo state, EncodingOptions encodingOp
             {
                 var hwType = encodingOptions.HardwareAccelerationType;
 
+                // AMD GPU has no MJPEG hw encoder
+                if (hwType == HardwareAccelerationType.vaapi
+                    && _mediaEncoder.IsVaapiDeviceAmd)
```

Comment:
Probably only whitelist Intel for this is better?

Reply:
make sense to me.